package/etc/conf.d/conflib/netsource/app-netsource-cisco_esa.conf

```diff
@@ -38,7 +38,8 @@ filter f_cisco_esa_authentication {
     or match('failed\s*authentication.', value("MESSAGE"))
     or match('Time\s*offset\s*from\s*UTC:', value("MESSAGE"))
     or match('[vV]ersion:\s+.*\s+SN:\s+.*', value("MESSAGE"))
-    or match('\s*login|logout:\s*[^\s]*\s*[Uu]ser:[\w\-]+\s*session:[^\s]+', value("MESSAGE"))
```

Comment:
@rjha-splunk @dhrpatel2-crest
I proposing instead of editing this condition use `match(old_condition) or match(new_condition)`. Because you know Cisco device very popular, I worry that we can break something that not covering new regex

Reply:
reverted changes based on review comments.